src/components/change-log.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+
+const htmlRegex = /<\/?[a-z][\s\S]*>/i;
+
+const submissionChangesTable = [
+  ['8/2020', 'Add "eligiblePopulationExclusion" validation for CQM measures', `For "category": "quality" AND "submissionMethod": "registry" AND "isRegistryMeasure": "false", add validation so that "eligiblePopulationExclusion" MUST be 0 or NULL.  Note: Per the Measure Specification, Registries should already have reduced the denominator for excluded populations.  See details here: <a href='https://cmsgov.github.io/qpp-submissions-docs/measurements#single-performance-rate-measurements' target='_blank' rel='noopener noreferrer'>cmsgov.github.io/qpp-submissions-docs/measurements#single-performance-rate-measurements</a>`],
```

Comment:
I'm curious why use string arrays here instead of structured objects?
```
{date: ...,
change: ...,
description: ...}
```

Reply:
less to type out.